zone/gm_commands/parcels.cpp

```diff
@@ -4,304 +4,488 @@
-				Chat::White,
-				fmt::format(
-					"Player {} could not be found in this zone.  Ensure you are in the same zone as the player.",
-					player_name
-				).c_str()
-			);
-			return;
-		}
+				item_id = link_body.item_id;
 
-		auto parcels = player->GetParcels();
-		if (parcels.empty()) {
-			c->Message(Chat::White, fmt::format("No parcels could be found for {}", player_name).c_str());
-			return;
+				augment_ids = {
+					link_body.augment_1,
+					link_body.augment_2,
+					link_body.augment_3,
+					link_body.augment_4,
+					link_body.augment_5,
+					link_body.augment_6
+				};
+			}
 		}
 
-		c->Message(Chat::Yellow, fmt::format("Found {} parcels for {}.", parcels.size(), player_name).c_str());
-		for (auto const &p: parcels) {
-			c->Message(
-				Chat::Yellow,
-				fmt::format(
-					"Slot [{:02}] has item id [{:10}] with quantity [{}].",
-					p.second.slot_id,
-					p.second.item_id,
-					p.second.quantity
-				).c_str()
-			);
-		}
-	}
-	if (is_add) {
-		if (arguments < 4) {
-			SendParcelsSubCommands(c);
-			return;
-		}
 
-		if (!Strings::IsNumber(sep->arg[3]) || !Strings::IsNumber(sep->arg[4])) {
-			SendParcelsSubCommands(c);
-			return;
-		}
+		std::string note;
+		uint16      note_index    = 5;
+		uint16      augment_index = 0;
 
-		auto to_name  = std::string(sep->arg[2]);
-		auto item_id  = Strings::ToUnsignedInt(sep->arg[3]);
-		auto quantity = Strings::ToUnsignedInt(sep->arg[4]);
-		auto note     = std::string(sep->argplus[5]);
+		if (arguments >= note_index) {
+			while (sep->IsNumber(note_index)) {
+				augment_ids[augment_index] = Strings::ToUnsignedInt(sep->arg[note_index]);
+
+				augment_index++;
+				note_index++;
+			}
+
+			note = sep->argplus[note_index];
+		}
 
 		auto send_to_client = CharacterParcelsRepository::GetParcelCountAndCharacterName(
 			database,
 			to_name
 		);
-		if (send_to_client.at(0).character_name.empty()) {
-			c->MessageString(Chat::Yellow, CANT_FIND_PLAYER, to_name.c_str());
+
+		const std::string& character_name = send_to_client.front().character_name;
+		const uint32 character_id = send_to_client.front().char_id;
+
+		if (character_name.empty()) {
+			c->MessageString(Chat::White, CANT_FIND_PLAYER, to_name.c_str());
 			return;
 		}
 
-		auto next_slot = c->FindNextFreeParcelSlot(send_to_client.at(0).char_id);
-		if (next_slot == INVALID_INDEX) {
+		const int free_slot = c->FindNextFreeParcelSlot(character_id);
+		if (free_slot == INVALID_INDEX) {
 			c->Message(
-				Chat::Yellow,
+				Chat::White,
 				fmt::format(
 					"Unfortunately, {} cannot accept any more parcels at this time. Please try again later.",
-					send_to_client.at(0).character_name
+					character_name
 				).c_str()
 			);
 			return;
 		}
 
+		const std::string& note_string = (
+			!note.empty() ?
+			fmt::format(
+				" with a note of '{}'",
+				note
+			) :
+			""
+		);
+
 		if (item_id == PARCEL_MONEY_ITEM_ID) {
-			if (quantity > INT32_MAX) {
+			if (quantity > std::numeric_limits<int>::max()) {
```

Comment:
Kingly, is there an advantage of using std::numeric_limit over the define in limits.h of INT32_MAX?  Curious if there is an advantage from one over the other.  Perhaps the compiler replaces them both anyways?